### PR TITLE
6.0.0.Beta

### PR DESCRIPTION
**Pull request by Jad-el-khoury**
6.0.0.Beta

### CODE REVIEW

**Comment by Jad-el-khoury** [comment]
I'm aiming for a 6.0.0 release. 
While waiting for EMO approval, instructions suggested making a beta release.
Should I really merge to master as suggested here?
https://github.com/eclipse/lyo/wiki/Lyo-Release-Process#phase-1a-new-version-git-push-and-eclise-maven-release-deploy

**Comment by berezovskyi** [comment]
Yes, this way you'll test the release to maven central.

**Review by berezovskyi — APPROVED** [review]

**Pull request merged**
